BannerlordTweaks.csproj

```diff
@@ -125,6 +125,7 @@
     <Compile Include="Settings.cs" />
     <Compile Include="SubModule.cs" />
     <Compile Include="Patches\TournamentBehaviourPatch.cs" />
+    <Compile Include="TweakDefaultCharacterDevelopmentModel.cs" />
```

Comment:
For consistency that should be TweakedCharacterDevelopmentModel.cs

Reply:
oh fk, that was a failure from my side... sigh

Reply:
No worries, it should be TweakedCharacterDevelopmentModel.cs instead of TweakedDefault

Reply:
DefaultCharacterDevelopmentModel is the model that I am overriding from Banenrlords